src/main/java/com/tstepnik/planner/domain/user/userregistrationdto/RegisterUserDto.java

```diff
@@ -0,0 +1,58 @@
+package com.tstepnik.planner.domain.user.userregistrationdto;
+
+import javax.validation.constraints.NotEmpty;
+
+public class RegisterUserDto {
```

Comment:
When creating classes related to specific entity I like to use this format of naming: EntityNamePurposeTheRest, which in case of this dto translates to: `UserRegiserDTO`. It is just easier for other developers to use later on, since they will start typing User... and intellisense will suggest all of User related classess.